prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-actions/adjudicator-actions.component.html

```diff
@@ -106,6 +106,12 @@
       <mat-icon>edit_calendar</mat-icon>
       <span>Change Date of Birth</span>
     </button>
+    <button mat-menu-item
+            *ngIf="(Role.MANAGE_ENROLLEE | inRole )&& canChangeToA()"
+            (click)="onChangeToA()">
+      <mat-icon>edit</mat-icon>
+      <span>Change ToA</span>
```

Comment:
"TOA" rather than "ToA".  Former used much more often in front-end

Reply:
Changed